R/gt_options_default.R

```diff
@@ -2,6 +2,7 @@ gt_options_default <- function() {
 
   dplyr::tribble(
     ~parameter,                           ~scss, ~category,             ~value,
+    "table_id",                          FALSE,  "table",               NA_character_,
```

Comment:
How does this connect to the rest of the PR?

Reply:
The table ID value is stored here. And it's later retrieved in the `as.tags.gt_tbl()` function.

Reply:
Ha! Correct. Missed it. Thanks